README.md

```diff
@@ -24,6 +24,8 @@ Check this [wiki](https://github.com/nimblehq/gin-templates/wiki/Directories) fo
 
 ## Usage
 
+- Add `GO111MODULE=auto` to shell config (`.bashrc` for Bash or `.zshrc` for ZSH).
```

Comment:
In `Go 1.16`, they dropped `GO111MODULE=auto` setting and it is now by default `GO111MODULE=on`(also in the upcoming [Go 1.17](https://blog.golang.org/go116-module-changes) Go will completely ignore `GO111MODULE `). So I think we should not use the old setting as `gin-templates` prerequisite is  `Go 1.16`. Also, this setting will affect others go projects also as we are charging shell config.

> no required module provides package github.com/nimblehq/gin-templates: go.Mod file not found in the current directory or any parent directory; see 'go help modules'

Regarding this error @Lahphim faced(it show in my machine also), its is because `go get github.com/nimblehq/gin-templates` command does not use old GOPATH way in `Go 1.16` and can't fine the downloaded repo(github.com/nimblehq/gin-templates) and shows the error.

In the newer module way, `go get` command stored  package in `$GOPATH/pkg/mod` with versioning. I think we can use this path while building the binary `go build -o $GOPATH/bin/nimble-gin module-path`.

Another way is to download old gopath way with out changing the global config(`.bashrc` / `.zshrc`)  is `O111MODULE=on go get github.com/nimblehq/gin-templates` but it will not work in upcoming [Go 1.17](https://blog.golang.org/go116-module-changes) as they plan to ignore `GO111MODULE `

So I think we can try to with newer module way to install/build the project. What do you think? @gutakk

Reply:
Thanks for pointing this out @junan . That makes sense.
I will close this PR and create the new task to update README with your suggestion.